src/serde_arrow_type.erl

```diff
@@ -12,17 +12,19 @@
     arrow_int/0,
     arrow_uint/0,
     arrow_float/0,
+    arrow_bin/0,
     erlang_type/0
 ]).
 
 -type arrow_type() ::
     arrow_bool()
     | arrow_int()
     | arrow_uint()
-    | arrow_float().
+    | arrow_float()
+    | arrow_bin().
 %% Any primitive logical type in Apache Arrow that is supported by `serde_arrow'.
 
--type arrow_bool() :: arrow_boolean.
+-type arrow_bool() :: b.
```

Comment:
```suggestion
-type arrow_bool() :: bool.
```

So arrow_bool -> bool, arrow_bin -> bin. And we keep only the Nx compatible ones really short.

Reply:
Changed.